src/presentation/connect/sign-msg.tsx

```diff
@@ -72,7 +73,7 @@ const ConnectSignMsg: React.FC<WithConnectDataProps> = ({ connectData }) => {
         password,
         networkFromString(network)
       );
-      await sendResponseMessage(true, signedMsg.signature);
+      await sendResponseMessage(true, signedMsg);
```

Comment:
Is this then printed as JSON in the response?

Reply:
Not JSON string just a JS object (but yeah this message is serialized in order to reach inject script if it's your question)